docker-compose.yml

```diff
@@ -30,8 +28,14 @@ services:
           nocopy: true
         read_only: true
     depends_on:
-      # Depends on node to get the volume data for nginx configuration
       - node
+      - update-nginx-conf
```

Comment:
why can't we keep the previous solution?

Reply:
Because the Nginx config is not updated automatically. Once we run `docker-compose up` for the first time the volume is created on the host and `node`'s content is copied to it. Next time we update Nginx config on `node` it's not reflected on the volume. The volume is mounted to the container and masks the new config instead.

Reply:
Thanks. I missed that.

Reply:
@andbortnik we already have `./run.sh` script which does a lot of node start related procedure. Let's put the copying there instead. It even will make more sense since without proper nginx configuration the node won't work correctly.

Reply:
I put it to `./run.sh` initially but then I thought Nginx config is out of context of the file. Also `./run.sh` can be used out of docker-compose configuration. So I think docker-compose configuration is a better place for that.